templates/emails/questionnaire

```diff
@@ -1,30 +1,35 @@
-<?=$values['weeks_ago']?> weeks ago we sent your letter to <?=$values['recipient_name']?>, your
-<?=$values['recipient_position']?>. (For reference, there's a copy of your
-letter at the bottom of this email)
+<?=$values['weeks_ago']?> weeks ago you sent a letter to <?=$values['recipient_name']?>, your
+<?=$values['recipient_position']?> through WriteToThem.com
 
-- If you HAVE had a reply (not just an acknowledgement), please
-click on the link below:
+For reference, there's a copy of your letter at the bottom of this email.
 
-    <?=$values['yes_url']?>
+Please help us understand how well representatives are doing at
+responding to their constituents.
 
-- If you HAVE NOT had any reply at all, OR you have only had an
-acknowledgement, please click on the link below:
+Did your <?=$values['recipient_position']?> reply to your message?
 
-    <?=$values['no_url']?>
+YES! - please click here:
+<?=$values['yes_url']?>
 
-If you feel that neither link is suitable, then please do not answer the
-questionnaire.
+YES, but I don't think my question was answered! - please click here:
+<?=$values['unsatisfactory_url']?>
 
-Your feedback will allow us to publish performance tables of the responsiveness
-of all the politicians in the UK. The majority of
-<?=$values['recipient_position_plural']?> deserve credit and respect for their
-conscientiousness as they respond promptly and diligently to the needs and
-views of <?=$values['their_constituents']?>. Likewise, we're keen to expose the
-minority of <?=$values['recipient_position_plural']?> who don't.
+YES, but it was only an acknowledgement and I haven't received anything
+since! - please click here:
+<?=$values['no_url']?>
 
-The letter you sent to your <?=$values['recipient_position']?> will be deleted
-from our database within the next two weeks. Our full privacy policy can be read here:
-    https://www.writetothem.com/about-qa#personal
+NO! - please click here:
+<?=$values['no_url']?>
+
+NO, but I didn't expect/require a response! - please click here:
+<?=$values['not_expected_url']?>
+
+Your feedback allows us to publish performance tables of the responsiveness
+of all the politicians in the UK. We are keen to give politicians the credit
```

Comment:
Does this email get sent to users who’ve written to their MEPs, MSPs, or local councillors?

(The `<?=$values['recipient_position']?>` at the top would suggest so.)

If so, do we still want to tell them their questionnaire response “allows us to publish performance tables”? Since the WTT Zeigeist only covers Westminster MPs.